docs/swarm/swarm-mode.md

```diff
@@ -73,10 +73,41 @@ To add a manager to this swarm, run the following command:
     192.168.99.100:2377
 ```
 
+### Configure the advertise address
+
+Manager nodes use an advertise address to allow other nodes in the swarm access
+to the API and overlay networking. The other nodes on the swarm must be able to
```

Comment:
API may be ambiguous; this is not the "remote API", but the "Swarm" API (do we have a name for that?)